.github/workflows/check-pg_search-schema-upgrade.yml

```diff
@@ -145,6 +145,7 @@ jobs:
         uses: actions/github-script@v7
         if: steps.generate_commit_message.outputs.schema_diff_detected == 'true'
         with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
```

Comment:
After adding `github-token`, the CI still doesn't work. Maybe the permission issue? @philippemnoel

Reply:
I'll take a look

Reply:
It should be env.GITHUB_TOKEN